framework/db/src/com/cloud/utils/db/Merovingian2.java

```diff
@@ -63,9 +63,7 @@
     private Merovingian2(long msId) {
         super(MerovingianMBean.class, false);
         _msId = msId;
-        Connection conn = null;
-        try {
-            conn = TransactionLegacy.getStandaloneConnectionWithException();
+        try (Connection conn = TransactionLegacy.getStandaloneConnectionWithException();) {
```

Comment:
Connection is used in ConnectionConcierge object. There will be failure somewhere else if Connection get GC'ed.